config/schema.json

```
@@ -6,12 +6,12 @@
   "type": "object",
   "properties": {
     "logLevel": {
-      "description": "Sets the log level for the `consul-ecs mesh-init` and `consul-ecs health-sync` commands. Defaults to `INFO`.",
+      "description": "Sets the log level for the `consul-ecs control-plane` and `consul-ecs health-sync` commands. Defaults to `INFO`.",
       "type": ["string", "null"],
       "enum": ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", null]
     },
     "bootstrapDir": {
-      "description": "The directory at which to mount the shared volume where Envoy bootstrap configuration is written by `consul-ecs mesh-init`.",
+      "description": "The directory at which to mount the shared volume where dataplane configuration is written by `consul-ecs control-plane`.",
```

minor: I think "dataplane" on its own can be ambiguous. Better to call it "consul-dataplane" or "Consul Dataplane" here.

```suggestion
      "description": "The directory at which to mount the shared volume where consul-dataplane configuration is written by `consul-ecs control-plane`.",
```